src/a_piece/mod.rs

```diff
@@ -0,0 +1,259 @@
+#[cfg(test)]
```

Comment:
These attributes should be on the module, not specific item. This whole module should be skipped unless compiling tests.